src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs

```diff
@@ -61,6 +61,22 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative)]
         internal static extern void Asn1StringFree(IntPtr o);
 
+        internal static string GetOidValue(SafeSharedAsn1ObjectHandle asn1Object)
+        {
+            Debug.Assert(asn1Object != null);
+
+            bool added = false;
+            asn1Object.DangerousAddRef(ref added);
+            try
+            {
+                return GetOidValue(asn1Object.DangerousGetHandle());
```

Comment:
Can you remind me why the other GetOidValue doesn't just take a SafeHandle and pass it to a P/Invoke declared to take a SafeHandle rather than an IntPtr?